integration_test/tests/util.rs

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Tests for methods found under the `== Util ==` section of the API docs.
+
+#![allow(non_snake_case)] // Test names intentionally use double underscore.
+#![allow(unused_imports)] // Because of feature gated tests.
+
+use bitcoin::hex::FromHex as _;
+use bitcoin::{Amount};
+use integration_test::{Node, NodeExt as _, Wallet};
+use node::vtype::*; // All the version specific types.
+
+#[test]
+fn utils__createmultisig__modelled() {
+    let node = Node::with_wallet(Wallet::Default, &[]);
+    node.fund_wallet();
+
+    // Use two new addresses to get public keys.
+    let addr1 = node.client.new_address().expect("new_address 1");
+    let addr2 = node.client.new_address().expect("new_address 2");
+    let info1 = node.client.validate_address(&addr1).expect("validateaddress 1");
+    let info2 = node.client.validate_address(&addr2).expect("validateaddress 2");
+
+    let pubkey1 = info1.address.as_ref().unwrap();
+    let pubkey2 = info2.address.as_ref().unwrap();
+
+    // For test purposes, just use the addresses as dummy pubkeys (in real use, use actual pubkeys).
+    let keys = vec![pubkey1.clone(), pubkey2.clone()];
+
+    let json: CreateMultisig = node.client.create_multisig(2, &keys, Some("legacy")).expect("createmultisig");
+    let model: Result<mtype::CreateMultisig, _> = serde_json::to_value(&json)
+        .and_then(|v| serde_json::from_value(v));
+    model.expect("CreateMultisig into model");
+}
+
+#[test]
+fn utils__estimatesmartfee__modelled() {
```

Comment:
Here you have the method marked as modellled which is correct. But in `verify/src/method/v17.rs` it is marked as returns nothing (my bug not yours). For all methods when removeing 'TODO' we have to check `verify` is correct because I made tons of mistakes there. Once you are done `verify` should pass.